src/shared/components/post/post-listing.tsx

```diff
@@ -760,7 +760,7 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
         {this.unreadCount && (
           <>
             {" "}
-            <span className="fst-italic">
+            <span className="badge text-bg-info">
               ({this.unreadCount} {I18NextService.i18n.t("new")})
             </span>
```

Comment:
I recall before we used the dull color, we colored the font instead of giving it a background. Maybe try that instead while keeping the italic?

Reply:
How about this:

<img width="741" alt="Screenshot 2023-07-04 at 1 26 10 PM" src="https://github.com/LemmyNet/lemmy-ui/assets/643417/a446f246-17a3-4e77-a10a-a6ef9e3269e0">
<img width="707" alt="Screenshot 2023-07-04 at 1 26 01 PM" src="https://github.com/LemmyNet/lemmy-ui/assets/643417/7663a669-79e9-4718-990f-937130fb9334">

Reply:
Which theme color is that? I was thinking of using the info color.

Reply:
That's "danger" -- "info" feels semantically correct, but because we're using these theme colors more or less arbitrarily, it leads to some potentially confusing clashes:

<img width="644" alt="Screenshot 2023-07-04 at 1 50 56 PM" src="https://github.com/LemmyNet/lemmy-ui/assets/643417/8adcc01c-f9b0-4fe7-9745-4bb04e44704c">
<img width="588" alt="Screenshot 2023-07-04 at 1 50 45 PM" src="https://github.com/LemmyNet/lemmy-ui/assets/643417/e37883fb-51f0-4c42-923b-b9c7477f3251">

Reply:
It looks like the UI used to use text-warning:
```
<span className="small text-warning">
    ({this.unreadCount} {i18n.t("new")})
</span>
```